index.html

```diff
@@ -0,0 +1,666 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+      integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
+      crossorigin="anonymous"
+    />
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+    rel="stylesheet">
+        <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <ul class="nav">
+      <div class="flex-1">
+
+     
+      <li class="nav-item">
+        <a class="nav-link" href="#"
+          >
+          <span class="material-icons nav-icon" >
+            menu
+            </span></a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="#">
+          <span>
+            <img src="/img/youtube.png" alt="" srcset="" /> <span>YouTube</span>
+          </span></a
+        >
+      </li>
+    </div>
+    <div class="flex-2">
+
```

Comment:
- [ ] You managed to do everything very well. However, there's something missing in the process of reviewing, please re-review your PR by following the next steps 🙏 😄 

1. 1st, you go to the Files Changed page from your PR
2. 2nd, and after you spot the changes you see that need to be fixed, in the very same spot (as this current comment is) you'll write your comment and click on the 'Start a review' button
3. 3rd, you'll go to the top-right corner 'Review changes' button, and then write your review summary
4. 4th, click the 'Submit review' button

This way, you'll have the review summary the first thing, then your [inline comments](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) down below it in a very organized and good-looking way 😄 👍